### PR TITLE
Feat: Update Design Tokens according to the current state of Figma tokens

### PR DESCRIPTION
**Pull request by crishpeen**
- Introduce brand color scale
- Switch to REM units
- Update typography settings
- Remove unused code

### CODE REVIEW

**Review by literat — APPROVED** [review]

File: packages/design-tokens/src/default/scss/_typography.scss [hunk 1]
```diff
@@ -1,48 +1,52 @@
...
-$font-size-6: 1.25rem !default;
-$font-size-7: 1.5rem !default;
-$font-size-8: 2rem !default;
-$font-size-9: 2.5rem !default;
-$font-size-10: 3rem !default;
-$font-size-11: 3.5rem !default;
-$font-size-12: 4rem !default;
-$font-size-13: 5rem !default;
+// Font Size
+$font-size-1: 0.625rem !default; //10px
```
Comment (adamkudrna) [inline packages/design-tokens/src/default/scss/_typography.scss:9]: `// 10px` might slightly improve legibility.

File: packages/design-tokens/src/default/scss/_colors.scss [hunk 2]
```diff
@@ -31,123 +41,85 @@ $danger-600: #c95771 !default;
 $danger-800: #af4c62 !default;
 
 // Brand
-$brand-primary: #132930 !default;
-$brand-secondary: #77a6c0 !default;
+$brand-primary: $brand-550 !default;
+$brand-secondary: $brand-300 !default;
```
Comment (adamkudrna) [inline packages/design-tokens/src/default/scss/_colors.scss:45]: What if `$brand-secondary` needs to be derived from a palette other than `$brand-X`? Like here, blue and orange:

obrazek

Don't we need another graded palette for that?

**Review by adamkudrna — APPROVED** [review]

File: packages/design-tokens/src/default/scss/_colors.scss [hunk 3]
```diff
@@ -31,123 +41,85 @@ $danger-600: #c95771 !default;
...
+    link-primary-hover: $action-link-primary-hover,
+    link-primary-active: $action-link-primary-active,
+    link-primary-disabled: $action-link-primary-disabled,
+    link-primary-visited: $action-link-primary-visited,
+    link-secondary-default: $action-link-secondary-default,
+    link-secondary-hover: $action-link-secondary-hover,
+    link-secondary-active: $action-link-secondary-active,
+    link-secondary-disabled: $action-link-secondary-disabled,
+    link-secondary-visited: $action-link-secondary-visited,
 );
```
Comment (ondrej-kaspar) [inline packages/design-tokens/src/default/scss/_colors.scss:123]: `!default` ?

File: packages/design-tokens/src/default/scss/_colors.scss [hunk 4]
```diff
@@ -173,158 +145,41 @@ $emotion-danger-disabled: $gray-200 !default;
...
+    warning-hover: $emotion-warning-hover,
+    warning-active: $emotion-warning-active,
+    warning-disabled: $emotion-warning-disabled,
+    warning-background: $emotion-warning-background,
+    danger-default: $emotion-danger-default,
+    danger-hover: $emotion-danger-hover,
+    danger-active: $emotion-danger-active,
+    danger-disabled: $emotion-danger-disabled,
+    danger-background: $emotion-danger-background,
 );
```
Comment (ondrej-kaspar) [inline packages/design-tokens/src/default/scss/_colors.scss:168]: `!default` ?

File: packages/design-tokens/src/default/scss/_colors.scss [hunk 5]
```diff
@@ -173,158 +145,41 @@ $emotion-danger-disabled: $gray-200 !default;
...
-        $icon-danger-active,
-        #fff,
-    ),
+    main-default: $border-main-default,
+    main-hover: $border-main-hover,
+    main-active: $border-main-active,
+    main-selected: $border-main-selected,
+    main-disabled: $border-main-disabled,
+    focus-default: $border-focus-default,
 );
```
Comment (ondrej-kaspar) [inline packages/design-tokens/src/default/scss/_colors.scss:185]: `!default` ?

File: packages/design-tokens/src/default/scss/_colors.scss (hunk 2 above)
Comment (crishpeen) [reply packages/design-tokens/src/default/scss/_colors.scss]: After discussion, we decided to use **color** scales on lvl 0 of config instead. Each product can set up their own and use them on 1st level of color config (brand, bg, text, etc.).

File: packages/design-tokens/src/default/scss/_colors.scss [hunk 6]
```diff
@@ -1,330 +1,189 @@
...
-$info-800: #78a9c5 !default;
+$glacier-300: #77a6c0 !default;
+$glacier-400: #678fa6 !default;
+$glacier-500: #57798c !default;
 
-$success-200: #f2f5e7 !default;
-$success-400: #b7d16d !default;
-$success-600: #9ebf40 !default;
-$success-800: #8fad3a !default;
+$porcelain-050: #f0f4f5 !default;
```
Comment (ondrej-kaspar) [inline packages/design-tokens/src/default/scss/_colors.scss:15]: Do you insist on leading zero? I would remove it.

**Review by adamkudrna — APPROVED** [review]

File: packages/design-tokens/src/default/scss/_colors.scss (hunk 6 above)
Comment (crishpeen) [reply packages/design-tokens/src/default/scss/_colors.scss]: Nope, removed.

**Pull request merged**